app/Http/Controllers/Api/UsersController.php

```diff
@@ -27,6 +27,19 @@ public function __construct(Registrar $registrar)
     {
         $this->registrar = $registrar;
         $this->transformer = new UserTransformer;
+
+        $this->middleware('auth.api');
```

Comment:
Probs. I can add to them as well. Was concentrating on the Users controller, but makes sense to add to all of them now.